setup.py

```diff
@@ -91,8 +91,16 @@
 
 install_requires = [
     'Click>=6.3',
+    'Flask-OAuthlib>=0.9.3',
+    'Flask>=0.11.1',
+    # https://github.com/inveniosoftware/invenio-oauth2server/pull/188
+    'oauthlib>=2.1.0,<3.0.0',
```

Comment:
I'd say we can merge first the one on `invenio-oauth2server`, less "unpinning" to be done later on.

Reply:
I agree! I wanted to have the tests passing to ensure I have done all the modifications, but I had in mind the same thing! I will ask today.
Have a nice night/day Esteban! :)

Reply:
> I'd say we can merge first the one on `invenio-oauth2server`, less "unpinning" to be done later on.

@egabancho so unfortunately I need to pin `requests-oauthlib` and `oauthlib` because the relesed version of `Flask-OAuthlib` does not set a max version for these 2... so Travis tests with min versions fail otherwise... :(

Reply:
I think we should put this requirements somewhere else, maybe [here](https://github.com/inveniosoftware/invenio-migrator/blob/master/setup.py#L49).
The idea behind the migrator is that it can be installed on both, Invenio v1 and v3, for v1 you just run `pip install invenio-migrator` and that's it, and I guess we don't need/want all this libraries installed also, since we are only going to dump records using Invenio v1 code.
For v3 you'll do something like `pip install invenio-migrator[loader]` (at least)
Did I make sense? ☺️

Reply:
> I think we should put this requirements somewhere else, maybe [here](https://github.com/inveniosoftware/invenio-migrator/blob/master/setup.py#L49).
> The idea behind the migrator is that it can be installed on both, Invenio v1 and v3, for v1 you just run `pip install invenio-migrator` and that's it, and I guess we don't need/want all this libraries installed also, since we are only going to dump records using Invenio v1 code.
> For v3 you'll do something like `pip install invenio-migrator[loader]` (at least)
> Did I make sense? ☺️

OH! You are right, I forgot about that! Thanks for the head up!!!
I tried again, what about this last commit?